build/VS2008/zstd/zstd.vcproj

```diff
@@ -518,6 +518,10 @@
 				RelativePath="..\..\..\lib\common\threading.h"
 				>
 			</File>
+			<File
+				RelativePath="..\..\..\programs\util.c"
```

Comment:
VS2010 was failing without the util.c being present in zstd project file. I added them in both to make them compile.

Reply:
Well, this script if for Visual Studio 2008 specifically, because it uses a different semantic than Visual 2010+.

And indeed, I second @felixhandte, it seems pretty strange to have `util.c` in the "header" section, rather than the "source files" section just above.

Note that the Visual compilation test on Appveyor passes because it only tests VS2015 compilation (and I do not know why, since this test was supposed to run multiple versions of Visual studio in the past, something to look at).

Anyway, bottom line : passing Appveyor test does not prove that VS2008 compilation works.